contrib/lang/ess/packages.el

```diff
@@ -140,11 +139,6 @@ not play nicely with autoloads"
 
 (when (configuration-layer/layer-usedp 'auto-completion)
   (defun ess/post-init-company ()
-    (spacemacs|add-company-hook ess-mode))
-
-  (defun ess/init-company-ess ()
-    (use-package company-ess
-      :if (configuration-layer/package-usedp 'company)
-      :defer t
-      :init
-      (push 'company-ess-backend company-backends-ess-mode))))
+    (spacemacs|add-company-hook ess-mode)
+    (push 'company-R-objects company-backends-ess-mode)
+    (push 'company-R-args company-backends-ess-mode)))
```

Comment:
Wouldn't adding the push statements (without the `spacemacs|add-company-hook`) cause an error if the user decided not to enable `company` or `auto-complete`?